### PR TITLE
Document ralph's tutorial

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

To improve accessibility and use of ralph tool, a tutorial oriented for non expert user is added to the current ralph documentation so that the project could get more external visibility.

## Proposal

Write backend-oriented tutorials

- [x] Introduction
- [ ] Database backend tutorial
- [ ] Logging backend tutorial
- [x] Storage backend tutorial
- [ ] Stream backend tutorial

### CODE REVIEW

**Comment by quitterie-lcs** [comment]
@SergioSim @jmaupetit It's a first proposal of what we could provide as a first-use tutorial. Tell me what you think about it and if I need to continue with this approach (one tutorial per input type backend) or if we need to be more generic

File: tutorial/storage.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+If you went that far, it means that you have:
+
+1. pushed and fetched archives from Swift
+2. Then extract logs from the archives, convert them and validate the conversion to a new format
+
+Yet, you have to finally pushed the successfully converted data to the new backend, Elasticsearch.
+
+With the complete following command, you will push directly from Swift
+to elasticsearch by converting your logs "on-the-fly".
```
Comment (quitterie-lcs) [inline tutorial/storage.md:131]:
```suggestion
to elasticsearch by converting your logs "on-the-fly".
```
Really not sure how to translate "à la volée" :grin:

Comment (jmaupetit) [reply tutorial/storage.md:131]: That's correct. :1st_place_medal:

File: tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
+# Learn how to use Ralph
+
+> This tutorial is intended for new or beginning users of `ralph`.
+> It is required to have some knowledge of the UNIX system flows, basic notions in data management.
```
Comment (jmaupetit) [inline tutorial/index.md:4]:
```suggestion
> It is required to have some knowledge of the UNIX system streams and basic notions in data management.
```

File: tutorial/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+> An understanding of edX and xAPI learning data standardization formats is preferred.
+
+_Nota : In this tutorial, it is considered your machine is on Linux distribution.
+If you are working on Windows, we highly recommend you either to use a machine on Linux
+or to install [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install)._
```
Comment (jmaupetit) [inline tutorial/index.md:9]:
```suggestion
> _nota bene_: in this tutorial, it is considered that you are running a UNIX system. If you are a Windows user, consider installing [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/install)._
```

File: tutorial/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Let's start !
```
Comment (jmaupetit) [inline tutorial/index.md:11]:
```suggestion
## Let's start!
```

**Comment by quitterie-lcs** [comment]
[Tutoriel utilisation de ralph](https://trello.com/c/x1YDPFlH/8-tutoriel-utilisation-de-ralph)

**Comment by quitterie-lcs** [comment]
Deprecated since new version of Ralph documentation will made tutorials integrated within the project's documentation

**Pull request closed without merging**